cmd/e2e_test.go

```diff
@@ -54,7 +54,7 @@ func init() {
 
 	klog.SetLogger(ginkgo.GinkgoLogr)
 
-	verbosity := 1
+	verbosity := 6
```

Comment:
Why is the verbosity changed? The commit message / PR description doesn't mention that.

Reply:
OK, thanks.